ocaml_emulator/platform_impl.ml

```diff
@@ -141,6 +141,9 @@ let set_dtc path =
     ( Printf.eprintf "Error accessing %s: %s\n%!" path (Unix.error_message e);
       exit 1)
 
+let set_dram_base mb =
+  dram_base_ref := Int64.of_int (int_of_string mb)
```

Comment:
I didn't use Arg.int because the user than have to input mem base address in Int. But with Arg.String I get the hex input in form of string and then convert hex to int using int_of_string function.
if `Int64.of_int (int_of_string mb)` is same as `Int64.of_string mb` I'll change that.

Reply:
I don't understand what your point is here